### ISSUE TITLE
Second set of navigation arrows

### ISSUE THREAD

**Issue by cmc333333** [18F/atf-eregs#505]
> Add “Previous” and “Next” navigation buttons at the top of the pages (i.e. arrows underneath “effective date”), instead of only at the bottom.

Should be simple, though we'll want to confirm this doesn't junk the interface up too much.

**Comment by tadhg-ohiggins** [comment]
So we're talking about something like this?
![image](https://user-images.githubusercontent.com/2626258/29337569-9b15c184-81c6-11e7-8219-35f9c776beb5.png)

**Comment by cmc333333** [comment]
That's my understanding. I'd appreciate a gut check from @micahtaylor and/or @carodew that it's not complicating the interface too much, and whether or not it makes sense to add to all clients or just ATF.

**Comment by carodew** [comment]
Oh wow I didn't think we had committed to doing this. I don't recommend proceeding because I think it makes it much more difficult to tell what is important in terms of visual hierarchy, this hasn't gone through any design or usability testing, and it's not based on user research.

**Comment by cmc333333** [comment]
Makes sense; I think this was the riskiest of the quick wins. Moving to the Abyss for later evaluation.

**Comment by carodew** [comment]
Ok, makes sense. 

Sorry if my previous response was curt, I was a little surprised. My main concern is that adding so much extra visual information in the UI can be really disruptive of the information hierarchy and can potentially confuse users who then don't have a clear visual path for what to look at, and we already have a lot of visual information on the screen. I've also not seen any user struggle with navigating the app.
